packages/perseus-editor/src/iframe-content-renderer.tsx

```diff
@@ -98,9 +98,9 @@ type Props = {
     // The URL that the iframe should load
     url: string;
     // The data-* suffix for passing information to the iframe's JS
-    datasetKey: string;
+    datasetKey?: string;
     // The value of the data-* attribute
-    datasetValue: any;
+    datasetValue?: any;
```

Comment:
`datasetValue` is going into a URL param, so it should probably be something that can get sensibly converted to a string.
```suggestion
    datasetValue?: string | number | boolean;
```